x2gbfs/providers/cantamen.py

```diff
@@ -148,6 +148,9 @@ def _all_bookees(self) -> Generator[Dict[str, Any], None, None]:
     def _all_places(self) -> Generator[Dict[str, Any], None, None]:
         places = self._load_response()['Place']
         for place in places:
+            name = place['Name']['Text']
```

Comment:
```suggestion
            name = place.get('Name',{}).get('Text','')
```
would handle cases where a place has no Name/Text. Your choice how reliable you deem the source.